operations/security_and_authentication/Audit_Logs.md

```diff
@@ -0,0 +1,114 @@
+# Audit Logs
+
+## Overview
+
+Audit logs are used to monitor the system and search for suspicious behavior.
+Host and Kubernetes API audit logging can be enabled to produce extra audit logs for analysis.
+Enabling audit logging is optional. If enabled it generates some load and data on the non-compute nodes \(NCNs\).
+
+By default, host and Kubernetes API audit logging are not enabled.
+It is not required for both to be enabled or disabled at the same time.
+
+## Enable or disable audit logging for host and Kubernetes APIs
+
+The method for updating the audit log settings varies depending on the state of the system.
+
+1. Select one of the following options based on the installation status of the system:
+
+   For each of the following options, only enable the desired level of audit logging.
+   It is not required to enable both.
+
+   * During CSM install, from the PIT node
+
+     To update the audit log settings during the installation, use one of the following options:
+
+     * **Option 1**
+
+       During the installation, audit logging is enabled or disabled by modifying the CSI settings.
+       To enable or disable audit logging, use the following flags with the `csi config init` command. For more information on using flags, see `csi config init -h`.
+
+       `ncn-mgmt-node-auditing-enabled`: Set to `true` to enable host logging or to `false` to disable host logging.
+
+       ```console
+       pit# csi config init --ncn-mgmt-node-auditing-enabled=true [other config init options]
+       ```
+
+       `k8s-api-auditing-enabled`: Set to `true` to enable Kubernetes API logging or to `false` to disable Kubernetes API logging.
+
+       ```console
+       pit# csi config init --k8s-api-auditing-enabled=true [other config init options]
+       ```
```

Comment:
````suggestion
       * `ncn-mgmt-node-auditing-enabled`

          Set to `true` to enable host logging or to `false` to disable host logging.

          ```console
          pit# csi config init --ncn-mgmt-node-auditing-enabled=true [other config init options]
          ```

       * `k8s-api-auditing-enabled`

          Set to `true` to enable Kubernetes API logging or to `false` to disable Kubernetes API logging.

          ```console
          pit# csi config init --k8s-api-auditing-enabled=true [other config init options]
          ```
````

Reply:
It seems cleaner to make these into two sub-bullets, to also indicate that they are independent of one another, not steps to be followed in order.